### PR TITLE
[bugfix/CMS-12825] - "Updated the list of supported libraries"

### PR DESCRIPTION
**Pull request by tgavane**

### CODE REVIEW

File: check_if_image_instrumentable.sh [hunk 1]
```diff
@@ -86,9 +86,12 @@ fi
...
+supported_libc=(glibc-2.26-33.amzn2.0.1.x86_64.rpm* glibc-2.26-33.amzn2.0.2.x86_64.rpm* libc-bin_2.19-0ubuntu6.13_amd64.deb* libc-bin_2.19-0ubuntu6.14_amd64.deb* libc-bin_2.19-0ubuntu6.15_amd64.deb* libc-bin_2.19-0ubuntu6.6_amd64.deb* libc-bin_2.19-0ubuntu6.7_amd64.deb* libc-bin_2.19-0ubuntu6.9_amd64.deb* libc-bin_2.19-18+deb8u10_amd64.deb* libc-bin_2.19-18+deb8u6_amd64.deb* libc-bin_2.19-18+deb8u7_amd64.deb* libc-bin_2.23-0ubuntu10_amd64.deb* libc-bin_2.23-0ubuntu11_amd64.deb* libc-bin_2.24-11+deb9u1_amd64.deb* libc-bin_2.24-11+deb9u3_amd64.deb* libc-bin_2.24-11+deb9u4_amd64.deb* libc-bin_2.27-3ubuntu1_amd64.deb* libc-bin_2.27-3ubuntu1.4_amd64.deb* musl-1.1.15-r8.apk* musl-1.1.16-r14.apk* musl-1.1.16-r15.apk* musl-1.1.18-r2.apk* musl-1.1.18-r3.apk* musl-1.1.19-r10.apk* musl-1.1.20-r3.apk* musl-1.1.20-r4.apk* musl-1.1.22-r3.apk* musl-1.1.24-r3.apk* musl-1.1.24-r9.apk* musl-1.1.24-r10.apk*
```
Comment (sakky016) [inline check_if_image_instrumentable.sh]: This is incorrect. The supported list of glibcs should not have the entries for instrumented version of glibcs (those having LI and QCRS suffixes)

File: check_if_image_instrumentable.sh [hunk 2]
```diff
@@ -86,9 +86,12 @@ fi
...
...4.rpm glibc-2.17-261LI.el7.centos.x86_64.rpm glibc-2.17-293LI.el7.centos.x86_64.rpm glibc-2.17-308QCRS.el7.centos.1.x86_64.rpm glibc-2.17-318QCRS.el7.centos.x86_64.rpm glibc-2.17-324QCRS.el7.centos.x86_64.rpm glibc-2.28-152QCRS.el8.centos.x86_64.rpm glibc-2.28-73QCRS.el8.centos.x86_64.rpm glibc-common-2.17-106LI.el7.centos.x86_64.rpm glibc-common-2.17-107LI.el7.centos.8.x86_64.rpm glibc-common-2.17-158LI.el7.centos.1.x86_64.rpm glibc-common-2.17-197LI.el7.centos.2.x86_64.rpm glibc-common-2.17-197LI.el7.centos.x86_64.rpm glibc-common-2.17-223LI.el7.centos.x86_64.rpm glibc-common-2.17-261LI.el7.centos.3.x86_64.rpm glibc-common-2.17-261LI.el7.centos.6.x86_64.rpm glibc-common-2.17-261LI.el7.centos.x86_64.rpm glibc-common-2.17-293LI.el7.centos.x86_64.rpm glibc-common-2.17-308QCRS.el7.centos.1.x86_64.rpm glibc-common-2.17-318QCRS.el7.centos.x86_64.rpm glibc-common-2.17-324QCRS.el7.centos.x86_64.rpm glibc-common-2.26-33.amzn2.0.1.x86_64.rpm glibc-common-2.26-33.amzn2.0.2.x86_64.rpm glibc-common-2.28-152QCRS.el8.centos.x86_64.rpm glibc-common-2.28-73QCRS.el8.centos.x86_64.rpm glibc-minimal-langpack-2.28-152QCRS.el8.centos.x86_64.rpm glibc-minimal-langpack-2.28-73QCRS.el8.centos.x86_64.rpm)
```
Comment (sakky016) [inline check_if_image_instrumentable.sh]: entries having `common `and `minimal `are not required.

**Review by sakky016 — CHANGES_REQUESTED** [review]
Please refer to individual comments.

File: check_if_image_instrumentable.sh (hunk 1 above)
Comment (tgavane) [reply check_if_image_instrumentable.sh]: Done!

File: check_if_image_instrumentable.sh (hunk 2 above)
Comment (tgavane) [reply check_if_image_instrumentable.sh]: Done!

**Review by sakky016 — COMMENTED** [review]
Looks good to me.

**Review by bsalunke — COMMENTED** [review]
Looks like we have update the list of supported libraries not just for alpine but for other libs as well. 
Please update commit message accordingly. 
"Updated the list of supported libraries"

**Review by bsalunke — APPROVED** [review]

**Pull request merged**